<launch>

  <arg name="tel_pub_rate" default="100" />

  <node name="teleop_node" pkg="bwrobot" type="teleop_server.py">
  	<param name="teleop_pub_rate" value="$(arg tel_pub_rate)" />
  </node>

</launch>
